system/Database/SQLite3/Forge.php

```diff
@@ -164,6 +153,33 @@ protected function _alterTable(string $alterType, string $table, $processedField
         }
     }
 
+    /**
+     * @param list<string>|string $columnNames column names to DROP
+     *
+     * @return bool
+     *
+     * @throws DatabaseException
+     */
+    public function dropColumn(string $table, $columnNames)
+    {
+        $sqlTable = new Table($this->db, $this);
+
+        $sqlExecuteResult = $sqlTable->fromTable($this->db->DBPrefix . $table)
+            ->dropColumn($columnNames)
+            ->run();
+
+        if ($sqlExecuteResult === false) {
+            $columns = is_array($columnNames) ? implode(',', $columnNames) : $columnNames;
+
+            throw new DatabaseException(
+                'Failed to drop column. Table: "' . $table
+                . '", Column: "' . $columns . '"'
+            );
```

Comment:
`dropColumn()` may return `false` when `$this->db->DBDebug` is `false`.
So make this the same, please.
https://github.com/codeigniter4/CodeIgniter4/blob/f60d27702b6844826343619755340b66faff9383/system/Database/Forge.php#L787-L800

Reply:
@ping-yee Can you complete this PR?
We are very close to the completion.

Reply:
@kenjis Sorry for the stagnation, I haven’t had much energy to deal with things other than work a while ago.
I modify this right now.

Reply:
Done.

Reply:
However, the rector removed the return false line, implying that if sqlExecuteResult is false, the method would still return false.